client/test/Foo.js

```diff
@@ -0,0 +1,22 @@
+import React, { PropTypes } from 'react';
+
+const propTypes = {};
+
+const defaultProps = {};
+
+class Foo extends React.Component {
```

Comment:
Please use a more descriptive name

Reply:
This is just a dummy test file, since I'm currently working on the UI, I"ll focus now and working on this test.

Reply:
@rafaelhernandez Fixed!